site/reference/pkg/update/README.md

```diff
@@ -67,33 +43,47 @@ VERSION:
 
 ```
 --strategy:
-  Controls how changes to the local package are handled.  Defaults to fast-forward.
-
-    * resource-merge: perform a structural comparison of the original /
-      updated Resources, and merge the changes into the local package.
-    * fast-forward: fail without updating if the local package was modified
+  Defines which strategy should be used to update the package. This will change
+  the update strategy for the current kpt package for future updates. If a
+  strategy is not provided, the strategy specified in the package Kptfile will
```

Comment:
`Kptfile` in verbatim ?

Reply:
This is also inside a code block, so already in verbatim.